test/lit/passes/j2cl-inline.wast

```diff
@@ -1,16 +1,16 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
 
-;; RUN: foreach %s %t wasm-opt --no-inline=*_@once@_* --optimize-j2cl --inlining --vacuum --optimize-level=3 -all -S -o - | filecheck %s
+;; RUN: foreach %s %t wasm-opt --no-inline=*_\<once\>_* --optimize-j2cl --inlining --vacuum --optimize-level=3 -all -S -o - | filecheck %s
```

Comment:
Looks like the test failed on the windows bot. Perhaps something with the new symbol is a problem for windows commandline operations? If so this might help:
```suggestion
;; RUN: foreach %s %t wasm-opt "--no-inline=*_\<once\>_*" --optimize-j2cl --inlining --vacuum --optimize-level=3 -all -S -o - | filecheck %s
```